.devcontainer/Dockerfile

```
@@ -0,0 +1,25 @@
+FROM ubuntu:focal
+
+WORKDIR /root
+
+RUN mkdir -p ./tools/scripts/lib_install
+ADD ./tools/scripts/lib_install ./tools/scripts/lib_install
```

Hadolint

[error] Use COPY instead of ADD for files and folders

(DL3020)